packages/frontend/components/authorize/AuthorizeView.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import BasicButton from '../basic/BasicButton.vue';
+
+
+interface Props {
+  username: string
+  applicationName: string
+  claims: string[]
+}
+
+
+withDefaults(defineProps<Props>(), {})
+
+
+</script>
+
+
+<template>
+  <div class="flex flex-row w-full min-h-screen h-fit">
+    <div class="w-50em sm:max-w-80% h-fit m-a flex flex-col items-center b-1 b-dark-300 rd-8px p-3 w-90% sm:p-10" >
```

Comment:
We want the app to be coded mobile first, `sm:max-w-80%` only applies to >= sm, not <= sm.

Once again, we want to use tailwind units instead of em, px etc. and we want to use the full class name `width-full` instead of `w-90%`.

Why exactly are we using specific percentages?  Seems like a bad practice.